src/transformers/trainer_callback.py

```diff
@@ -77,6 +83,9 @@ class TrainerState:
     epoch: Optional[float] = None
     global_step: int = 0
     max_steps: int = 0
+    logging_steps: int = 500
+    eval_steps: int = None
```

Comment:
Nit: here I'm not sure we will actually get to `None`, so you could put 500 so the type hint is correct.